src/priorityq.py

```diff
@@ -0,0 +1,44 @@
+"""Implement Priority Queue."""
+
+from priority_heap import Priority_Heap
+
+class Priority_Q():
+    def __init__(self):
+        self.heap = Priority_Heap()
+
+    def pop(self):
```

Comment:
I get an infinite loop when I run my tests on your priorityq. I think the underlying problem is with the implementation of your Heap it goes into an infinite loop when you try and pop on a heap with this structure [1,1,2,2,3,3,4,4,5,5]